jwt-authorizer/src/builder.rs

```diff
@@ -54,6 +54,26 @@ where
         }
     }
 
+    pub fn from_jwks(path: &str) -> AuthorizerBuilder<C> {
+        AuthorizerBuilder {
+            key_source_type: KeySourceType::JwksPath(path.to_owned()),
+            refresh: Default::default(),
+            claims_checker: None,
+            validation: None,
+            jwt_source: JwtSource::AuthorizationHeader,
+        }
+    }
+
+    pub fn from_jwks_text(text: &str) -> AuthorizerBuilder<C> {
```

Comment:
It could, but then it would be inconsistent with the pattern of other functions like `from_ec_pem_text` (which creates a  `KeySourceType::ECString` ) ; which I feel would be more inconsistent for end-user. Happy to change though if you feel otherwise

Reply:
Yeah, you are right!